code/modules/mob/living/basic/space_fauna/spider/giant_spider/giant_spiders.dm

```diff
@@ -343,6 +343,7 @@
 
 	AddElement(/datum/element/wall_tearer)
 	AddElement(/datum/element/web_walker, /datum/movespeed_modifier/below_average_web)
+	AddElement(/datum/element/mech_prying)
```

Comment:
you need to add mech_prying.dm to the dme

Reply:
Oh pffft. Whoops